auth/user_mgt.go

```diff
@@ -333,7 +339,7 @@ func (u *UserToUpdate) validatedRequest() (map[string]interface{}, error) {
 			if err != nil {
 				return nil, err
 			}
-			req["mfaInfo"] = mfaInfo
+			req["mfa"] = multiFactorEnrollments{mfaInfo}
```

Comment:
nit: Can you link to the docs https://cloud.google.com/identity-platform/docs/reference/rest/v1/accounts/update for easy reference next time? Thanks!

Reply:
Super nit: Please add this link as a code comment if possible.

Reply:
This change is not showing up in the diff yet.. otherwise lgtm

Reply:
Have added the link as you suggested @prameshj . Please take a look